lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -866,6 +866,10 @@ SwiftLanguageRuntime::GetNumFields(CompilerType type,
       return rti->getNumFields();
     }
   }
+  case TypeInfoKind::Builtin: {
+    // Clang types without debug info may present themselves like this.
+    return {};
```

Comment:
are the included tests exercising this case? if so, which ones hit this?

Reply:
Yes, the new test runs into this.